EIPS/eip-6366.md

```diff
@@ -228,10 +228,10 @@ Needs discussion.
 
 ## Reference Implementation
 
-First implementation of EIP-6366 could be found here:
+First implementation of [EIP-6366](https://github.com/orochi-network/EIP-6366) could be found here:
 
-- [ERC-6366 Core implementation](https://github.com/orochi-network/EIP-6366/blob/main/contracts/ERC6366/ERC6366Core.sol)
-- [ERC-6366 Meta implementation](https://github.com/orochi-network/EIP-6366/blob/main/contracts/ERC6366/ERC6366Meta.sol)
+- [EIP-6366 Core implementation](https://github.com/orochi-network/EIP-6366/blob/main/contracts/EIP-6366/EIP-6366-Core.sol)
+- [EIP-6366 Meta implementation](https://github.com/orochi-network/EIP-6366/blob/main/contracts/EIP-6366/EIP-6366-Meta.sol)
```

Comment:
If the files are renamed as I suggested in the previous PR we have to change the links